src/pages/Cart/CartPage.jsx

```diff
@@ -15,7 +16,9 @@ export const CartPage = () => {
     <div className="main-content-container">
       <h2 className="heading">My Cart</h2>
       {cartItem.length === 0 ? (
-        <ErrorMessage message={"your cart is empty"} />
+        <Link to="/productpage">
+          <ErrorMessage message={"your cart is empty"} />
+        </Link>
```

Comment:
Suggestion, You can use NavLink, if there is a need to use ClassName.

Reply:
tried but it gives an error.will try after